src/server/models/news/router.js

```diff
@@ -0,0 +1,47 @@
+const router = require('express').Router();
+
+const path = require('path');
+const fs = require('fs');
+const util = require('util');
+const News = require('./model');
+
+const readFileAsync = util.promisify(fs.readFile);
+const writeFileAsync = util.promisify(fs.writeFile);
+
+const storageFilePath = path.resolve(__dirname, '../data.json');
+
+router.route('/api/my-news/:id?')
+    .get((req, res, next) => {
+        if (req.params.id !== undefined) {
+            News.findById(req.params.id)
+                .then((data) => {
+                    res.json(data)
+                })
+        }
+        News.find({userId: req.user._id})
+            .then((data) => {
+                res.json(data)
+            })
+    })
+    .delete((req, res, next) => {
+        News.remove({ _id: req.params.id })
+            .then((data) => res.json(data))
+            .catch(next);
+    });
+
+router.route('/api/my-news')
```

Comment:
post = for adding new news
get = get all news

Reply:
put here isn't correct

Reply:
fixed